### ISSUE TITLE
react-native-maps/example requires peer of react-native@>=0.30.0 but none installed.

### ISSUE THREAD

**Issue by esutton** [lelandrichardson/react-native-maps#489]
To help new users get started and avoid these issues:
```
cd example
npm install
npm WARN react-native-maps@0.7.1 requires a peer of
 react-native@>=0.30.0 but none was installed.
```

Suggest change /example/package.json to match the react-native-maps/package.json:
```
  "dependencies": {
    "react": ">=15.2.0",
    "react-native": ">=0.30.0",
    "react-native-maps": "../"
  },
```

And add the draw over permission to AndroidManifest.xml so example does not crash on first launch.

`    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
`

**Comment by the-simian** [comment]
I can say at this exact moment - we should all (react native folks) be using React 15.2.1 . react-native hasn't caught up to 15.3.x just yet, and if you do decide to use that your console will get flooded with stylesheet warnings

**Comment by spikebrehm** [comment]
This should be fixed in [0.8.0](https://github.com/lelandrichardson/react-native-maps/releases/tag/v0.8.0). Feel free to re-open this issue if not.
